.gitignore

```diff
@@ -164,3 +164,4 @@ pip-log.txt
 
 # Mac crap
 .DS_Store
+/.vs/**/*
```

Comment:
So maybe move this upward in the file to the Visual Studio section, just for clarity.